src/SearchInput/SearchInput.tsx

```diff
@@ -19,6 +19,7 @@ export interface SearchInputProps extends CommonFieldTypes {
   onFound: (element: string) => void
   resultsRelativePosition?: boolean
   showIcon?: boolean
+  onBlur?: (e: FormEvent<HTMLInputElement>) => void
```

Comment:
I think the right typing would be:
```suggestion
  onBlur?: (e: FocusEvent<HTMLInputElement>) => void
```

Reply:
We are using form event everywhere else tho 🤔

Reply:
Should I change it across all instances?

Reply:
MDN actually says it's a FocusEvent so I'll update it

Reply:
The typing is probably wrong yes, I believe for `onFocus` and `onBlur`, it should be a `FocusEvent`